src/data/constants.ts

```diff
@@ -16,6 +16,11 @@ export const BASE_URL = 'https://google.com';
 
 export const API_URL = 'https://google.com/api';
 
+export const REACT_APP_GOOGLE_CLIENT_ID = '548504225330-akr3ktdrpt5cn50262bdoagafumgtjdu.apps.googleusercontent.com';
+export const REACT_APP_API_AUTH_URL = 'https://dev-teamcyperecom.onrender.com/api/v1/auth/google-login';
+export const REACT_APP_API_AUTH_SIGNUP_URL = 'https://dev-teamcyperecom.onrender.com/api/v1/auth/google';
+export const REACT_APP_API_BASE_URL = 'https://dev-teamcyperecom.onrender.com';
```

Comment:
```suggestion
```
@SergeInGithub Store these values in `.env` This is not good practice to store sensitive within the codebase.